scripts/release_scripts/update_changelog_and_credits.py

```diff
@@ -447,6 +449,39 @@ def create_branch(
             target_branch, current_release_version_number))
 
 
+def get_release_summary_lines():
+    """Returns the lines from release summary file. It checks whether
+    incorrect email is present or ordering of sections is invalid.
+    In either case, the user will be asked to update the release
+    summary file and the lines will be re-read.
+    """
+    is_invalid_email_present = True
```

Comment:
In the Oppia codebase, don't start variables with verbs. Use invalid_email_is_present, ordering_is_invalid instead.

Reply:
Done.